Cesium.IntegrationTests/stdlib/io/fprintf.c

```diff
@@ -0,0 +1,38 @@
+#include <stdio.h>
+#include <stdint.h>
+#include <inttypes.h>
+
+int main(void)
+{
+    const char* s = "Hello";
+    printf("Strings:\n"); // same as puts("Strings");
+    printf(" padding:\n");
+    printf("\t[%10s]\n", s);
+    printf("\t[%-10s]\n", s);
+    printf("\t[%*s]\n", 10, s);
+    printf(" truncating:\n");
+    printf("\t%.4s\n", s);
+    printf("\t%.*s\n", 3, s);
+
+    printf("Characters:\t%c %%\n", 'A');
+
+    printf("Integers:\n");
+    printf("\tDecimal:\t%i %d %.6i %i %.0i %+i %i\n",
+        1, 2, 3, 0, 0, 4, -4);
+    printf("\tHexadecimal:\t%x %x %X %#x\n", 5, 10, 10, 6);
+    printf("\tOctal:\t\t%o %#o %#o\n", 10, 10, 4);
+
+    printf("Floating point:\n");
+    printf("\tRounding:\t%f %.0f %.32f\n", 1.5, 1.5, 1.3);
```

Comment:
I would comment out %f.32 format for now.

Reply:
Yeah, let's comment it out for now.